conans/requirements.txt

```diff
@@ -1,7 +1,7 @@
 requests>=2.25, <3.0.0
 urllib3>=1.26.6, <1.27
 colorama>=0.3.3, <0.5.0
-PyYAML>=3.11, <=6.0
+PyYAML==6.0.1
```

Comment:
I am concerned of too restrictive constraint would break some special users that might workaround the cython issue, use older PyYAML version in some special distros or whatever. I have done https://github.com/conan-io/conan/pull/14323 to propose allowing future PyYAML patches too, I think it should be pretty safe and better in general than a strong pinning to 6.0.1

Reply:
Okiii, we can decline this PR then, i guess we'll have to wait for PyYAML to fix their builds :disappointed:

Reply:
Thank you!

Reply:
It will, i don't question that, i just hope that PyYAML will resolve issues with their builds. Because if they wont, this issue will come up again sometime :)